purge.py

```diff
@@ -1,15 +1,22 @@
 #!/usr/bin/python3
+
 """ purge.py
     MediaWiki Action API Code Samples
-    Demo of `purge` module: Sending post request to purge cache of one page
+    Demo of `purge` module: Sending post request to  purge first 10 pages in the main namespace
     MIT license
 """
+
 import requests
+
 URL = 'https://en.wikipedia.org/w/api.php'
+
 PARAMS = {"action": "purge",
           "generator": "allpages",
```

Comment:
There should be four spaces only used here, looks like you are using 8 or something. See sample code for `hello.py` used in the API:Documentation template below: 

```#!/usr/bin/python3

"""
    hello.py

    MediaWiki Action API Code Samples
    Demo of `...` module
    MIT license
"""

import requests

S = requests.Session()

URL = "https://en.wikipedia.org/w/api.php"

PARAMS = {
    'action':"query",
    'format':"json"
}

R = S.get(url=URL, params=PARAMS)
DATA = R.json()

print(DATA)
```